content/03.methods.md

```diff
@@ -64,14 +64,26 @@ Alignments were futher processed using following the Broad Institute's Best Prac
 Duplicates were marked using Samblaster[@doi:10/f6kft3] v0.1.24, BAMs merged and sorted using Sambamba [@doi:10/gfzsfw] v0.6.3.
 Lastly, resultant BAMs were processing using Broad's Genome Analysis Tool Kit (GATK) [@url:https://software.broadinstitute.org/gatk/] v4.0.3.0, BaseRecalibrator submodule.
 
-### Germ Line Single Nucleotide Variant Calling
+### Quality Control of Sequencing Data
+NGSCheckmate [doi:10.1093/nar/gkx193] was peformed on matched tumor/normal crams to confirm sample matches and remove mis-matched samples from the dataset. 
+Cram inputs were preprocessed using bcftools to filter and call snps using default parameters[@url:https://github.com/parklab/NGSCheckMate] and the resulting VCFs were used to run NGSCheckmate using [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) in the D3b GitHub repository. 
```

Comment:
```suggestion
Cram inputs were preprocessed using bcftools to filter and call SNPs using default parameters[@url:https://github.com/parklab/NGSCheckMate] and the resulting VCFs were used to run NGSCheckmate using [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) in the D3b GitHub repository. 
```

Reply:
I think we call these SNVs elsewhere - if so, we probably want to update this. Also, the link at the D3b github is probably better.

Reply:
```suggestion
CRAM inputs were preprocessed using bcftools to filter and call snps using default parameters[@url:https://github.com/parklab/NGSCheckMate] and the resulting VCFs were used to run NGSCheckmate using [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) in the D3b GitHub repository. 
```
because this is a file type, correct?

Reply:
hmm, I usually call them SNPs when talking about germline and SNVs when talking about somatic

Reply:
Are you looking at common variants across the population here? If so, I think that's the point to use `polymorphism`. Otherwise, I think `variant` is probably preferred. It wasn't clear to me that this was a point where the subject was focused on common variants. Maybe this means that SNP is still preferred but the sentence could be revised for clarity.